src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.searchForm {
+  margin-top: 20px;
+}
+
+.searchPic {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  padding-top: 25px;
+  padding-left: 26px;
+  padding-bottom: 26px;
+}
+
+.searchPicSmall {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  padding-top: 15px;
+  padding-left: 13px;
+  padding-bottom: 16px;
+}
+
+.searchBar {
+  padding: 0;
+  outline: none;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir-Book, sans-serif;
+}
+
+.searchBarBig {
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.searchBarSmall {
+  padding-left: 33px;
```

Comment:
2 point [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
now you've created 3 classes, let's use only two